docs/examples/Irrigation.ipynb

```diff
@@ -2,37 +2,47 @@
  "cells": [
```

Comment:
<p>Instead of hardcoding lat/lon. Can we use <code>grdc_latitude</code>  and <code>grdc_longitude</code>  vars instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>check</p>